installing/installing_with_agent/installing-with-agent.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+[id="installing-with-agent"]
+= Installing with Agent
+include::_attributes/common-attributes.adoc[]
+:context: installing-with-agent
+
+toc::[]
+
+== Agent installation
+Agent installation is based on the technology of Assisted Installer (AI), that is link:https://console.redhat.com[console.redhat.com] and the Infrastructure Operator. It is a command line tool that generates an image that runs the Assisted Service and the Assisted Installer Agent.
+//TODO>>include telemetry GA
+
+It is a subcommand that resides in the {product-title} installer and also utilizes the {product-title} installer for the input asset management.
+
+This subcommand generates a live {op-system-first} ISO with an embedded Ignition config containing `systemd` units to start the Assisted service, Agent, network, etc. that are running in containers pulled from the {product-title} release payload.
```

Comment:
It also contains a service that starts the cluster installation if all the required hosts have registered with assisted-service and passed validation.  @rwsu  please share the related jira.

Reply:
`start-cluster-installation.service` is the systemd service that does this job.

Jira: [AGENT-38](https://issues.redhat.com/browse/AGENT-38) which is part of epic [AGENT-10](https://issues.redhat.com/browse/AGENT-10)

Systemd service: https://github.com/openshift/installer/blob/agent-installer/data/data/agent/systemd/units/start-cluster-installation.service
Script run by the service: https://github.com/openshift/installer/blob/agent-installer/data/data/agent/files/usr/local/bin/start-cluster-installation.sh.template

Reply:
In my opinion, I think there is no harm to giving details on implementation. I worked on a jira for adding more conceptual information for the Horizontal Pod Autoscaler, because we had feedback from customers/TAMS etc that instead of referring to our docs.openshift , users were referring to the specific Kubernetes page as there was more conceptual information.  @celebdor wdyt?

Reply:
If we want to provide some conceptual information about how the system works, it should be enough to say that "the live ISO runs all of the components to orchestrate the installation" or something like that, without listing the individual parts or systemd units (we don't want users to somehow come to rely on that level of implementation detail). It is definitely useful to mention that the containers are pulled from the release payload, since that will be relevant for someone managing an image mirror.

Reply:
ok that makes sense, will incorporate that.